packages/styles/src/link/index.css

```
@@ -0,0 +1,82 @@
+/**
+ * Link styles
+ * @variant {small | medium | large} syn-link The size of the link
+ * @boolean { false } syn-link--quiet Applies the quiet link styling
+ * @boolean { false } syn-link--disabled Applies the disabled link styling
+ */
+.syn-link {
+  align-items: center;
+  color: var(--syn-link-color);
+  display: inline-flex;
+  font: var(--syn-body-medium-regular);
+  gap: var(--syn-spacing-x-small);
+}
+
+/* Use inline text styling when there is no syn-icon */
+.syn-link:not(:has(syn-icon)) {
+  display: inline;
+}
```

I would do it the other way round:

```suggestion
.syn-link {
  color: var(--syn-link-color);
  font: var(--syn-body-medium-regular);
}

/* Use inline text styling when there is no syn-icon */
.syn-link:has(syn-icon) {
  display: inline-flex;
  align-items: center;
  gap: var(--syn-spacing-x-small);
}
```

---

I don´t have a strong opinion on this, I am fine with it either way, so just changed it to your suggestion.